backend/src/v5/models/tickets.js

```diff
@@ -32,14 +33,17 @@ const determineTicketNumber = async (teamspace, project, model, type) => {
 	return (lastTicket?.number ?? 0) + 1;
 };
 
-Tickets.addTicket = async (teamspace, project, model, ticket) => {
+Tickets.addTicket = async (teamspace, project, model, template, ticketData, isFederation) => {
 	const _id = generateUUID();
-	const number = await determineTicketNumber(teamspace, project, model, ticket.type);
-	await DbHandler.insertOne(teamspace, TICKETS_COL, { ...ticket, teamspace, project, model, _id, number });
+	const number = await determineTicketNumber(teamspace, project, model, ticketData.type);
+	const ticket = { ...ticketData, teamspace, project, model, _id, number };
+	await DbHandler.insertOne(teamspace, TICKETS_COL, ticket);
+	publish(events.MODEL_NEW_TICKET,
+		{ teamspace, project, model, ticketData: serialiseTicket(ticket, template), isFederation });
```

Comment:
don't serialise the ticket

It's up to the event handler to do this, as you can't be sure the data isn't designed to be used internally

Reply:
I'd say don't bother passing down the template, get the event manager to pull it and serialise it there